internal/olm/operator/registry/index/registry_pod.go

```diff
@@ -102,6 +101,10 @@ func (rp *RegistryPod) init(cfg *operator.Configuration) error {
 
 	// TODO(rashmi/venkat): If FBCPath is empty, assign it a default value (maybe something thats hosted on a registry)
 
+	if rp.FBCPath == "" {
+		return fmt.Errorf("error: FBC path cannot be empty")
+	}
+
```

Comment:
If the FBC directory is empty, shouldn't we return an error? This scenario means there is no FBC to inject the bundle into, which should be a failure case, right?

In what case would the FBC directory be empty?

Reply:
Outdated, disregard.